app/models/manageiq/providers/openstack/cloud_manager/vm/remote_console.rb

```diff
@@ -13,7 +13,7 @@ def remote_console_acquire_ticket(_console_type, _proxy_miq_server = nil)
       return nil, url, nil, nil, nil, 'novnc_url', nil
     end
 
-    def remote_console_acquire_ticket_queue(protocol, userid, proxy_miq_server = nil)
+    def remote_console_acquire_ticket_queue(protocol, userid)
```

Comment:
When you call queue_object_action in the api controller code, does it go here or directly to the remote_console_acquire_ticket method?